ci/android.groovy

```diff
@@ -15,15 +15,15 @@ def uploadArtifact() {
 
 def compile(type = 'nightly') {
   common.tagBuild()
-  def gradleOpt = ''
+  def gradleOpt = "-PbuildUrl='${currentBuild.absoluteUrl}' "
   if (type == 'release') {
-    gradleOpt = "-PreleaseVersion=${common.version()}"
+    gradleOpt = "-PreleaseVersion='${common.version()}'"
```

Comment:
it will looks the build URL in that case, right?

Reply:
oh! good point! I'll fix it